civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.pages_default.inc

```diff
@@ -30,6 +30,8 @@ function civihr_employee_portal_features_default_page_manager_pages() {
       ),
     ),
     'logic' => 'and',
+    'type' => 'none',
```

Comment:
This is just adding missing defaults or is it making any real change?

Reply:
just adding missing defaults

(I really hate how things like this happens everytime we update features :( )